spec/unit/crypto/cross-signing.spec.js

```diff
@@ -669,6 +675,7 @@ describe("Cross Signing", function() {
         const bobDeviceTrust2 = alice.checkDeviceTrust("@bob:example.com", "Dynabook");
         expect(bobDeviceTrust2.isVerified()).toBeFalsy();
         expect(bobDeviceTrust2.isTofu()).toBeFalsy();
+        alice.stopClient();
```

Comment:
will these stops be called in the case of failure? `expect` tends to throw an internal exception when things go wrong.